data/transport missions.txt

```diff
@@ -40,6 +40,37 @@ mission "Immigrant Workers"
 
 
 
+mission "Vintage Spaceman"
+	description "A young man wants to find love on <destination>."
+	minor
+	passengers 1
+	to offer
+		random < 50
+	source "X"
+	destination
+		attributes core
+	on offer
+		conversation
+			`"Dude!" A patron in his mid twenties approaches after you accidentally make eye contact. "That your ship?" He continues. "You gotta help me. I'm in love!"`
+			`	He's dressed in a vintage spacesuit, something they wore before space travel was common. Useless these days, but a rare collector's item.`
+			`	"It all started last week. I was in the Spritney Beers chat room and started talking to this rad chick. We talk every day now. It's like she was made for me. I need to see her. She invited me to the next Spritney Beers concert on <system>. You have to take me!"`
+			choice
+				`	"Fine, but I'm in control of the music."`
+					accept
+				`	(Point like there's someone famous behind him and make a run for the door.)`
+					decline
+	on complete
+		payment 10000
+		conversation
+			`"We're here!" Vintage spaceman slams the emergency unlock and jumps out while your ship is still a meter off the ground. He runs off in to the crowd.`
+			`	You grab a drink, climb on top of the <ship> and relax.`
```

Comment:
Not everyone would do this. It should be a choice. (I would just got on with unloading/loading the ship.)